hoomd/mpcd/Integrator.cc

```diff
@@ -69,8 +69,9 @@ void mpcd::Integrator::update(unsigned int timestep)
         m_gave_warning = true;
         }
 
+    bool check_collide = checkCollide(timestep);
     // remove any leftover virtual particles
```

Comment:
I like to keep code that belongs together grouped under comments:
```suggestion
    // remove any leftover virtual particles
    bool check_collide = checkCollide(timestep);
```